src/nvim/lua/vim.lua

```diff
@@ -253,6 +253,54 @@ local function fn_index(t, key)
 end
 local fn = setmetatable({}, {__index=fn_index})
 
+local function __rpcrequest(...)
```

Comment:
do we need double-underscore? How does it differ from single-underscore?

Reply:
This was just from the  #8326 PR :) Soon it will be `vim.rpcrequest`.

Reply:
1.